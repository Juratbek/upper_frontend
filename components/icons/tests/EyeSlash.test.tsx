import { render, screen } from '@testing-library/react';

import { EyeSlashIcon } from '../EyeSlash';

describe('EyeSlash icon', () => {
  it('renders the icon', () => {
    render(<EyeSlashIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
