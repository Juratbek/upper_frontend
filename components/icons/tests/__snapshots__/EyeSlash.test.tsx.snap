// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EyeSlash icon > renders the icon 1`] = `
<svg
  role="icon"
  viewBox="0 0 256 256"
>
  <path
    d="M50.96,37.3092A4,4,0,1,0,45.04,42.69006L67.5874,67.49231c-38.25329,21.458-54.54358,57.30957-55.24267,58.8833a3.9975,3.9975,0,0,0,0,3.249c.33886.7627,8.51269,18.88672,26.82715,37.19922,16.957,16.95606,46.13378,37.168,88.82812,37.168a122.655,122.655,0,0,0,53.05518-11.68506L205.04,218.69006A4,4,0,1,0,210.96,213.3092Zm98.14246,119.84961a35.99216,35.99216,0,0,1-48.12354-52.936ZM128,195.99182c-31.97656,0-59.88965-11.64648-82.96387-34.61719a135.61312,135.61312,0,0,1-24.59277-33.376c3.64423-7.22461,20.08423-36.8003,52.67383-54.42383L95.57446,98.278a43.99336,43.99336,0,0,0,58.94141,64.83545l20.89307,22.98242A115.06293,115.06293,0,0,1,128,195.99182Zm115.65527-66.36719c-.40625.91211-10.19238,22.584-32.37695,42.4541a4.00008,4.00008,0,1,1-5.33789-5.959,136.40463,136.40463,0,0,0,29.61621-38.12109,135.68983,135.68983,0,0,0-24.59375-33.38282C187.88867,71.64123,159.97656,59.99182,128,59.99182a121.903,121.903,0,0,0-20.02344,1.62988,4,4,0,0,1-1.31836-7.89062A129.8525,129.8525,0,0,1,128,51.99182c42.69531,0,71.87109,20.21582,88.82812,37.17578,18.31543,18.31641,26.48829,36.44531,26.82715,37.208A3.9975,3.9975,0,0,1,243.65527,129.62463Zm-108.876-36.9873a4.00025,4.00025,0,0,1,1.49609-7.85938,44.15619,44.15619,0,0,1,35.53223,39.08692,3.99885,3.99885,0,0,1-3.61035,4.35449c-.127.01172-.252.01758-.377.01758a3.99965,3.99965,0,0,1-3.97754-3.62793A36.11549,36.11549,0,0,0,134.7793,92.63733Z"
    fill="black"
  />
</svg>
`;
